### ISSUE TITLE
Configure nick for MUC

### ISSUE THREAD

**Issue by resLotus** [pazznetwork/ngx-chat#71]
Hi, im new to xmpp chat and started using the library to program my application.
Im doing a multi user chat so Im primarily using the multiUserChatComponent plugin

My problem is that I want to change the visible nick for a user
Lets say that I have a user with jid: user-6@localhost
When the user sends a message in the chat it will appear as being sent by user-6
![image](https://user-images.githubusercontent.com/48839912/128545645-45cbbe5f-3ccb-4af5-a731-2d9d2761c1d6.png)

But I want that user to have nick asociated like for user-6 I want it to be Jhon Doe, so instead of saying user-6 it says Jhon Doe

I tried a few endpoints directly from ejabberd to set the nick and also tried the method modify member list from multi user chat component to see if it changed the nick

But nothing appears to be working

In short, my question is how to set the nick for a user in a multi user chat?

Thanks in advance for the responses

**Comment by CodeBast4rd** [comment]
To avoid some overhead we didn't implement for now a way to manage jids and nicks separately. You can of course use another source of information (another db or backend) to map the jids in the frontend to names or nicks. Or you can change the nicks manually with the ejabberd admin api or shell. 

But if you want this functionality to become part of the registration plugin a pull request is welcome.

**Comment by CodeBast4rd** [comment]
Nicks can be now provided through your own registration UI.

See:
https://github.com/pazznetwork/ngx-chat/tree/alternative-xmpp-adapter
